mis_builder/models/accounting_none.py

```diff
@@ -0,0 +1,173 @@
+# -*- encoding: utf-8 -*-
+
+"""
+>>> 1 + 1
+2
+
+>>> 1 + AccountingNone
+1
+
+>>> AccountingNone + 1
+1
+
+>>> AccountingNone + None
+AccountingNone
+
+>>> +AccountingNone
+AccountingNone
+
+>>> -AccountingNone
+AccountingNone
+
+>>> -(AccountingNone)
+AccountingNone
+
+>>> AccountingNone - 1
+-1
+>>> 1 - AccountingNone
+1
+>>> AccountingNone - None
+AccountingNone
+
+>>> AccountingNone / 2
+AccountingNone
+
+>>> 2 / AccountingNone
+Traceback (most recent call last):
+ ...
+ZeroDivisionError
+
+>>> AccountingNone // 2
+AccountingNone
+
+>>> 2 // AccountingNone
+Traceback (most recent call last):
+ ...
+ZeroDivisionError
+
+>>> AccountingNone * 2
+AccountingNone
+
+>>> 2 * AccountingNone
+AccountingNone
```

Comment:
I would say 2 * AccountingNone and AccountingNone * 2 should return 0. Because there was "something" in the expression, so AccountingNone dilutes.